Automattic/jetpack
# New bash script for updating composer.json versions for Jetpack packages

This script modifies the composer.json file of whatever directory it is run. 
It will update any packages prefixed with `automattic/jetpack-`, to point to the latest version according to its latest git tag. 

Probably will be most useful in the release scripts that branch off, since we:
a. Want to ship Jetpack with specific versions of packages
b. Want to preserve @dev in master branch

#### Changes proposed in this Pull Request:

*See above

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* New internal feature

#### Testing instructions:






- `yarn version-packages`: Should update all of the package versions at the root directory of Jetpack's composer.json
- `cd packages/ && ../bin/version-packages.sh`: Should get an error about needing composer.json in the directory 
- `cd packages/sync && ../../bin/version-packages.sh`: Should update just the composer.json found in `packages/sync/`
- In all cases, check to see if the composer.json files have been updated as expected. 

It also accepts a `--no-update` parameter if you only want to update composer.json for some reason. 

**New!**
- Accepts a `--package` parameter, which can be used for updating the package versions within a sub-package. For example `--package=sync --no-update` will update the composer.json in `JETPACK/packages/sync/composer.json`

#### Proposed changelog entry for your changes:

*
